solr/solr-ref-guide/src/solr-upgrade-notes.adoc

```diff
@@ -85,11 +85,18 @@ For more information about how to use this, see the section <<exporting-result-s
 
 * The `stats`, `facet`, and `timeseries` expressions now support percentiles and standard deviation aggregations.
 
-*Deprecations* 
+*Restricting file paths*
+
+Several Solr APIs allow specifying a file system location. This includes core creation, backup, restore and others. Prior to Solr 8.6 these parameters were not validated, and Solr would allow any absolute or relative path. From 8.6 we by default allow only paths that are relative to `SOLR_HOME`, `SOLR_DATA_HOME` and `coreRootDir`. If you need to create a core or store a backup outside these pre-allowed paths, you now need to tell Solr about what paths to allow.
+
+Solr can be configured with a comma separated list of paths to allow in `solr.xml` using the new `allowPaths` element, see <<format-of-solr-xml.html#the-solr-element,Solr.xml Parameters>>. When using the `solr.xml` file shipping with 8.6, you can configure the list of paths to allow through the system property `solr.allowPaths`. Please see `bin/solr.in.sh` or `bin\solr.in.cmd` for example usage. Using the value `*` will allow any path as in erlier versions.
+
+Windows SMB shares on the UNC format, such as `\\myhost\myshare\mypath` are now always disallowed. Please use drive letter mounts instead, i.e. `S:\mypath`.
 
-* Cross Data Center Replication (CDCR), in its current form, is deprecated and is scheduled to be removed in 9.0. Please refer to SOLR-14022.
```

Comment:
@chatman You had two *Deprecations* headlines in your upgrade note, and I remove the extra one in this commit

Reply:
See https://github.com/apache/lucene-solr/blob/master/solr/solr-ref-guide/src/solr-upgrade-notes.adoc and you'll understand (I'm removing the boldface *Deprecations* word from the end of the CDCR bullet